core/commonKotlin/src/LocalDate.kt

```diff
@@ -82,6 +82,13 @@ public actual class LocalDate actual constructor(public actual val year: Int, pu
             return LocalDate(yearEst, month, dom)
         }
 
+        public actual fun fromEpochDays(epochDays: Long): LocalDate {
+            require(epochDays in MIN_EPOCH_DAY..MAX_EPOCH_DAY) {
+                "Invalid date: boundaries of LocalDate exceeded"
+            }
+            return fromEpochDays(epochDays.toInt())
+        }
```

Comment:
```suggestion
        internal actual fun fromEpochDays(epochDays: Long): LocalDate =
            fromEpochDays(epochDays.clampToInt())
```
First, `clampToInt`, like above. Second: `public` seems incorrect.